test_othermlpackages/conda-xgboost.yaml

```diff
@@ -7,7 +7,7 @@ dependencies:
 - pip>=19.1.1
 - pytest
 - pytest-cov
-- xgboost
+- xgboost>=1.7.0
 - pandas<1.4 # Capped for xgboost only
```

Comment:
Do you know if the pandas cap is still required if we set xgboost >=1.7.0?

Reply:
Let's remove it and see what happens...

Reply:
Still works, keeping the change 🙂

Reply:
Awesome. Feel free to merge then if you agree with your own PR ;)